atd-vze/src/helpers/map.js

```diff
@@ -1,5 +1,7 @@
 import React from "react";
 import { Source, Layer } from "react-map-gl";
+import { isDev } from "./environment";
```

Comment:
funny to see this come up immediately after you mentioned it yesterday 😅

Reply:
😅 heh yep, it was top of mind since we mostly use it in this map code! now we are down to 2 uses in the app. 👍